### PR TITLE
Fix boolean defaults.

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fallout of ansible/ansible#72699.

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

**Comment by felixfontein** [comment]
@ndswartz @nerzhul @nibalizer @ovcharenko @pyykkis @rambleraptor @ravibhure @rosmo @russoz @sac @scathatheworm @schmots1 @scottanderson42 @sebasmannem @sgargan @smashwilson @srvg @tastychutney @tbouvet @tcraxs @timuster @troy2914 @xen0l

**Comment by felixfontein** [comment]
This needs to be merged first before any of the removal PRs can be merged, since we also need the fixes for the removed stuff in the stable-1 branch (where it's still there).

File: plugins/modules/web_infrastructure/django_manage.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def main():
 
             apps=dict(default=None, required=False),
             cache_table=dict(default=None, required=False, type='str'),
-            clear=dict(default=None, required=False, type='bool'),
+            clear=dict(default=False, required=False, type='bool'),
```
Comment (felixfontein) [inline plugins/modules/web_infrastructure/django_manage.py:273]: All options in `specific_boolean_params` are only tested for being truthy, so `None` and `False` are teated the same way.

File: plugins/modules/storage/glusterfs/gluster_peer.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def call_peer_commands(self):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            force=dict(type='bool', required=False),
+            force=dict(type='bool', required=False, default=False),
```
Comment (felixfontein) [inline plugins/modules/storage/glusterfs/gluster_peer.py:144]: Force is only tested for truthy in https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/storage/glusterfs/gluster_peer.py#L91, so `False` and `None` are treated the same way.

File: plugins/modules/cloud/rackspace/rax_clb.py [hunk 3]
```diff
@@ -276,7 +276,7 @@ def main():
             timeout=dict(type='int', default=30),
             type=dict(choices=['PUBLIC', 'SERVICENET'], default='PUBLIC'),
             vip_id=dict(),
-            wait=dict(type='bool'),
+            wait=dict(type='bool', default=False),
```
Comment (felixfontein) [inline plugins/modules/cloud/rackspace/rax_clb.py:279]: `wait` is passed on into `cloud_load_balancer`, and is there only tested for being truthy:
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L225
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L236
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L255

Therefore `False` and `None` are treated the same way.

**Review by felixfontein — COMMENTED** [review]
Three places are potentially problemated; I explained here why they are ok. This should make it easier to verify that this PR is ok.

File: plugins/modules/storage/netapp/na_cdot_license.py [hunk 4]
```diff
@@ -31,11 +31,13 @@
     description:
     - Remove licenses that have no controller affiliation in the cluster.
     type: bool
+    default: false
```
Comment (lonico) [inline plugins/modules/storage/netapp/na_cdot_license.py:34]: this file is deprecated.

**Review by lonico — APPROVED** [review]
approved for na_cdot file


In general, we use a value of None to detect whether the value was set by the user.

File: plugins/modules/storage/netapp/na_cdot_license.py (hunk 4 above)
Comment (felixfontein) [reply plugins/modules/storage/netapp/na_cdot_license.py:34]: Yes. It still exists (and will keep existing on the stable-1 branch), and I don't want sanity tests start randomly once the ansible-test PR is merged.

**Review by resmo — APPROVED** [review]

**Comment by resmo** [comment]
verified influx_db params

**Review by Andersson007 — APPROVED** [review]
postgresql_set - should work, thanks @felixfontein

**Review by gundalow — APPROVED** [review]
Thanks for doing this

File: plugins/modules/web_infrastructure/django_manage.py (hunk 1 above)
Comment (russoz) [reply plugins/modules/web_infrastructure/django_manage.py:273]: Personally I think it is better to remove it, but other than that, using a default that is actually in the domain of values for the specified type does look better.

**Comment by felixfontein** [comment]
@lonico @resmo @Andersson007 @gundalow thanks a lot for reviewing and merging!

**Pull request merged**